1.0/docs/start/first-element/step-3.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 3: Use data binding and properties"
+subtitle: Build your first Polymer element
+---
+
+
+Right now, the element doesn't do much. In this step, you'll give it a basic
+API, allowing you to configure the icon from markup, using an attribute, or
+from JavaScript, using a property.
+
+First, a bit of data binding. Find the `<iron-icon>` element and change the value of the `icon` attribute from `"polymer"` to  "`[[toggleIcon]]`".
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+&lt;!-- local DOM goes here -->
+&lt;iron-icon icon="[[toggleIcon]]">
+&lt;/iron-icon>
+</pre>
+
+Key information:
+
+  * `toggleIcon` is a <em>property</em> you'll define on the toggle button element. It doesn't have a default value
+yet.
+  * The `icon="[[toggleIcon]]" `assignment is a <em>data binding</em>. It links your element's `toggleIcon` <em>property</em> with the `<iron-icon>`'s `icon` property.
+
+You could now use your element and set the `toggleIcon` property in markup or using JavaScript, as shown in the following examples. <em>(You don't need to add this code to your project.)</em>
```

Comment:
I hear you, but I think the big change would have some downsides (you'd end up writing the demo code yourself, for one). And I'd have to go back and rewrite the codelab as well.  So I think I'm going to go with option #1.

In Q2, I'm hoping we can redo this flow as a fully-interactive step-by-step tutorial. Then we can do away with some of the fugly mechanics.